net-libs/libaccounts_glib/libaccounts_glib-1.27.recipe

```diff
@@ -0,0 +1,107 @@
+SUMMARY="Accounts management library for GLib applications"
+DESCRIPTION="This project is a library for managing accounts which can be used from GLib \
+applications. It is part of the [accounts-sso project][accounts-sso]."
+HOMEPAGE="https://gitlab.com/accounts-sso/libaccounts-glib"
+COPYRIGHT="2024 Accounts SSO"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://gitlab.com/accounts-sso/libaccounts-glib/-/archive/VERSION_1.27/libaccounts-glib-VERSION_1.27.tar.bz2"
+CHECKSUM_SHA256="e178c103e60ca34777afba94019a1c4571aedf9e54291b0faca71e5cad0628af"
+SOURCE_DIR="libaccounts-glib-VERSION_1.27"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libaccounts_glib$secondaryArchSuffix = $portVersion
+	lib:libaccounts_glib$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgio_2.0$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libsqlite3$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libaccounts_glib${secondaryArchSuffix}_devel = $portVersion
+	devel:libaccounts_glib$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	libaccounts_glib$secondaryArchSuffix == $portVersion base
+	"
+
+PROVIDES_tools="
+	libaccounts_glib${secondaryArchSuffix}_tools = $portVersion
+	cmd:ag_backup = $portVersion
+	cmd:ag_tool = $portVersion
+	"
+REQUIRES_tools="
+	libaccounts_glib$secondaryArchSuffix == $portVersion base
+	$REQUIRES
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	pygobject_python310
+	pygments_python310
+	devel:libcheck$secondaryArchSuffix
+	devel:libgio_2.0$secondaryArchSuffix
+	devel:libgirepository_1.0$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgobject_2.0$secondaryArchSuffix
+	devel:libsqlite3$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:gtkdoc_scan
+	cmd:meson
+	cmd:ninja
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python3
+	cmd:vapigen
+	cmd:xmllint
+	"
+
+BUILD()
+{
+	meson \
+		--buildtype=release \
+		--prefix=$prefix \
+		--libdir=$libDir \
+		--libexecdir=$binDir \
+		--datadir=$dataDir \
+		--includedir=$includeDir \
+		-D py-overrides-dir=$libDir/python3.10/vendor-packages \
+		build
+	ninja -C build
+}
+
+INSTALL()
+{
+	ninja -C build install
+
+	mkdir -p $dataDir
+	mv $prefix/share/gtk-doc $dataDir
```

Comment:
Doesn't documentation belong to $docDir, or somewhere else in $documentationDir, instead of $dataDir?

Reply:
gtk-doc is in $dataDir for all those packages installed here (can't find any references in $docDir for it.

Reply:
For this I'm holding off on changes, think we should make some agreement on this as it's used in multiple places.